src/ComponentBase.ts

```diff
@@ -400,7 +406,7 @@ export abstract class ComponentBase<P extends {}, S extends _.Dictionary<any>> e
 
     // Search Subscription "keyPropertyName" in Component props(this.props)
     private _findKeyFromPropertyName(props: Readonly<P>, keyPropertyName: keyof P): string {
-        const key = _.get(props, keyPropertyName);
```

Comment:
This doesn't work for old school resub users where keyProperty name can be "x.y.z" and it does [x][y][z].

Reply:
I know we had this in super early versions of ReSub, but keyPropertyName doesn't support this. I suppose this would 'break' if someone wasn't respecting the typings provided by ReSub
```keyPropertyName?: keyof P;```